### PR TITLE
Add args extraBuildInputs

### PR DESCRIPTION
**Pull request by GTrunSec**
Fixd some packages missing env such as LIBSVM(gcc9)

Also, nixpkgs-fmt (formatter) here.

image

### CODE REVIEW

**Comment by thomasjm** [comment]
Adding the new arg sounds good to me, but could you remove all the whitespace/formatting changes?

**Comment by GTrunSec** [comment]
Sorry, The nixpkgs-fmt is enabled in my editor automatically. 
I'm going to force re-commit.

File: templates/common.nix [hunk 1]
```diff
@@ -16,7 +16,8 @@
 
   # Arguments
   makeWrapperArgs ? "",
-  precompile ? true
+  precompile ? true,
+  extraBuildInputs
```
Comment (thomasjm) [inline templates/common.nix:20]: This should have `? []` after it, no?

File: templates/default.nix [hunk 2]
```diff
@@ -36,4 +36,6 @@ callPackage ./common.nix {
   # By default, it will just put the new depot at the end of JULIA_DEPOT_PATH.
   # You can add additional flags here.
   makeWrapperArgs = "";
+  # Extra Pkgs to precompile environment
```
Comment (thomasjm) [inline templates/default.nix:39]: Could you add a newline before this? And remove the space between the brackets to be consistent with the rest please :)

**Review by thomasjm — CHANGES_REQUESTED** [review]
Thanks for the PR, just a couple comments

File: templates/default.nix (hunk 2 above)
Comment (GTrunSec) [reply templates/default.nix:40]: thanks, please review again

**Comment by thomasjm** [comment]
Thanks! Looking at this I'm wondering if we should merge this option with the `extraLibs` variable already present in `default.nix`. `extraLibs` is used to set `LD_LIBRARY_PATH` at runtime whereas the option you added is used when building the depot. Not sure if a single option would make sense, would it work for your use case?

**Comment by GTrunSec** [comment]
Yes, We should merge the option with extra libs.  the lib.makeLibraryPath and lib.makeBinPath nothing different to set package in common option.

**Comment by thomasjm** [comment]
What do you think of c078539e5a51ab875da845d73b9f6db754d0bf42 ? It sets a default but leaves the ability to configure them independently.

**Comment by GTrunSec** [comment]
it does matter. the Binpath and  LibPath enough for us. Until this time it works well. Thanks

**Pull request merged**